soc/arm/silabs_exx32/Kconfig

```diff
@@ -26,6 +26,30 @@ config SOC_PART_NUMBER
 	  that you should not set directly. The part number selection choice defines
 	  the default value for this string.
 
+choice SOC_DCDC_GECKO_MODE
+	prompt "DC/DC mode"
+	depends on SOC_DCDC_GECKO
+	help
+	  Select power configuration mode of the on chip DC/DC converter.
+
+	config SOC_DCDC_GECKO_MODE_UNCONFIGURED
+		bool "Initial / Uconfigured"
+
+	config SOC_DCDC_GECKO_MODE_ON
+		bool "DC/DC On"
+
+	config SOC_DCDC_GECKO_MODE_OFF
+		bool "DC/DC Off"
+
+	config SOC_DCDC_GECKO_MODE_BYPASS
+		bool "Bypass"
+endchoice
+
+config SOC_DCDC_GECKO
```

Comment:
This option should select `HAS_EMU`. However `HAS_EMU` should be better renamed to `SOC_GECKO_EMU`.